tools/cldr-code/src/main/java/org/unicode/cldr/util/ICUServiceBuilder.java

```diff
@@ -40,6 +40,40 @@ public class ICUServiceBuilder {
         iso.setTimeZone(utc);
     }
 
+    private final CLDRFile cldrFile;
+    private final CLDRFile collationFile;
+
+    /**
+     * This private constructor is meant to be called only by ICUServiceBuilder.forLocale.
+     *
+     * @param cldrFile the CLDRFile
+     */
+    private ICUServiceBuilder(CLDRFile cldrFile) {
+        this(cldrFile, false /* locIsFake */);
+    }
+
+    /**
+     * This constructor is meant to be called only by the private constructor (with locIsFake =
+     * false) and by unit tests with fictional locale ID such as "xx" or "mul" (with locIsFake =
+     * true) for which ICUServiceBuilder.forLocale won't work
```

Comment:
```suggestion
     * false) and by unit tests with fictional locale ID such as "mul" (with locIsFake =
     * true) for which ICUServiceBuilder.forLocale won't work
```

drop reference to xx, don't use it.